index.html

```diff
@@ -120,6 +120,34 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+        <!-- ________ Nilasi J starts Here ________  -->
+        <div class="card">
+          <p class="name">Nilasi Jayathilake</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="" target="_blank">Jas</a>
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/NilasiJayathilake/" target="_blank">Nilasi Jayathilake</a>
+          </p>
+          <p class="about">Well here I am, just another CS student. I just want to get started with open Source projects and contribute.
+          I sort of like doing these things. I have a lot planned, Hopefully I stop being so adhd about it.</p>
+          <div class="resources">
+            <p>Things to do</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">Build a Portfolio</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Second Resource">Contribute to OS projects</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Third resource">Earn</a>
+              </li>
+            </ul>
```

Comment:
This `resources` section is for providing any useful websites or references that the fellow developers can use for any development based things. You can provide any resources that you frequently refer to. Some examples could be `Geeks for Geeks` or `YouTube tutorials`. 

Also note that you should provide the links inside the `href` attribute.